python/src/opendp/_convert.py

```diff
@@ -286,6 +292,23 @@ def _slice_to_string(raw: FfiSlicePtr) -> str:
     return ctypes.cast(raw.contents.ptr, ctypes.POINTER(ctypes.c_char_p)).contents.value.decode()  # type: ignore[reportOptionalMemberAccess,union-attr]
 
 
+def _bitvector_to_slice(val: Sequence[Any]) -> FfiSlicePtr:
+    if np is not None and isinstance(val, np.ndarray):
+        val = val.tobytes()
+    
+    if not isinstance(val, (bytes, bytearray)):
+        raise TypeError("Expected type is BitVector but input data is not bytes or bytearray.")
+
+    array = (ctypes.c_uint8 * len(val)).from_buffer_copy(val) # type: ignore[operator]
+    return _wrap_in_slice(array, len(val))
+
+
+def _slice_to_bitvector(raw: FfiSlicePtr) -> bytes:
+    n_bytes = (raw.contents.len + 4) // 8
+    buffer = ctypes.cast(raw.contents.ptr, ctypes.POINTER(ctypes.c_uint8))[0:n_bytes] # type: ignore
```

Comment:
You're right, technically it should be 7, not 4. However, since the input is always padded, any number from 0 and 7 would work. I've adjusted to use ceiling division (which is equivalent to 7).